<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDataItem</class>
 <widget class="QWidget" name="TestDataItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1543</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>设备名称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="name"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>型号规则：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mode"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>出厂编号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="number">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>委托单位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="tasknumber"/>
          </item>
          <item>
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>使用部门：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dept"/>
          </item>
          <item>
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>检定员：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="verification"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>检定日期:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="select">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(3, 58, 132);
color:white;
</string>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
            <property name="icon">
             <iconset resource="res/res.qrc">
              <normaloff>:/image/Index/u2189.svg</normaloff>:/image/Index/u2189.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="update">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(207, 132, 33);
color:white;
</string>
            </property>
            <property name="text">
             <string>重置</string>
            </property>
            <property name="icon">
             <iconset resource="res/res.qrc">
              <normaloff>:/image/Index/u2188.svg</normaloff>:/image/Index/u2188.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true">background-color:white;</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">Font-size:25px;</string>
            </property>
            <property name="text">
             <string>检定数据详情</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(207, 132, 33);
color:white;
</string>
            </property>
            <property name="text">
             <string>编辑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(3, 58, 132);
color:white;
</string>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:grey;
color:white;</string>
            </property>
            <property name="text">
             <string>返回</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="styleSheet">
          <string notr="true">background-color:white;</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="topMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="topMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="label_33">
               <property name="text">
                <string>检定基本信息：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="tableWidgetdevice"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="styleSheet">
             <string notr="true">background-color:white;</string>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>委托单位：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="customer_name">
               <property name="cursor">
                <cursorShape>IBeamCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>依据技术文件：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="based_documents">
               <property name="cursor">
                <cursorShape>IBeamCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>检定日期：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLineEdit" name="trace_date">
               <property name="cursor">
                <cursorShape>IBeamCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
             <item row="0" column="6">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>环境温度：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="7">
              <widget class="QLineEdit" name="temperature">
               <property name="cursor">
                <cursorShape>IBeamCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
             <item row="0" column="8">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>环境湿度：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="9">
              <widget class="QLineEdit" name="humidity">
               <property name="cursor">
                <cursorShape>IBeamCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>检定有效期至：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="measure_valid_date">
               <property name="cursor">
                <cursorShape>IBeamCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>检定员：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="NAME">
               <property name="cursor">
                <cursorShape>IBeamCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>介质：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QLineEdit" name="purchase_date">
               <property name="cursor">
                <cursorShape>IBeamCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>目前缺少</string>
               </property>
              </widget>
             </item>
             <item row="1" column="6">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>结论：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="7">
              <widget class="QLineEdit" name="conclusion">
               <property name="cursor">
                <cursorShape>IBeamCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
             <item row="1" column="8">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>限用说明：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="9">
              <widget class="QLineEdit" name="restriction_instruction">
               <property name="cursor">
                <cursorShape>ForbiddenCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>检定地点：</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="measure_address">
               <property name="cursor">
                <cursorShape>ForbiddenCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #f0f0f0; 
color: grey;</string>
               </property>
               <property name="text">
                <string>XXX</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="topMargin">
              <number>3</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_30">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>检定结果信息：</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_4">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>50</horstretch>
                   <verstretch>100</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color:rgb(3, 58, 132);
color:white;
</string>
                 </property>
                 <property name="text">
                  <string>历史修改记录</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>外观及功能性检查：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="appearance_true">
                 <property name="text">
                  <string>合格</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="appearance_false">
                 <property name="text">
                  <string>不合格</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>说明：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="appearance_remark"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="tableWidgetItem"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
